src/Lower.cpp

```diff
@@ -1905,10 +1906,10 @@ Stmt lower(Function f, const Target &t, const vector<IRMutator *> &custom_passes
     s = skip_stages(s, order);
     debug(2) << "Lowering after dynamically skipping stages:\n" << s << "\n\n";
 
-    if (t.has_feature(Target::OpenGL)) {
-        debug(1) << "Injecting OpenGL texture intrinsics...\n";
-        s = inject_opengl_intrinsics(s);
-        debug(2) << "Lowering after OpenGL intrinsics:\n" << s << "\n\n";
+    if (t.has_coordinates_feature()) {
```

Comment:
I think has_feature(OpenGL) | has_feature(Renderscript) is better here than adding a has_coordinates_feature to Target.